Updating Light and Dark themes to match Visual Studio

jmatthiesen opened the pull request:
Fixes VSTS #1052169

With this update:
* Light & Dark editor themes are updated to match Visual Studio with enhanced colors (barring some bug fixes we need to make in the editor).
* The existing Light & Dark editor themes are renamed to "Legacy - Light" and "Legacy - Dark"
* The Visual Studio Light theme is removed, now that Light contains the same colors.

sandyarmstrong commented:
@jmatthiesen is it intentional that you also added Light+/Dark+ files but did not add them to the csproj?

jmatthiesen commented:
> @jmatthiesen is it intentional that you also added Light+/Dark+ files but did not add them to the csproj?

Nope, good catch @sandyarmstrong  - I just removed these.

JoeRobich commented on main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Highlighting/themes/DarkStyle.json:
```diff
@@ -101,150 +90,156 @@
+		{ "name": "Keyword(Operator)", "fore": "keyword-blue" },
+		{ "name": "Keyword(Selection)", "fore": "keyword-blue" },
+		{ "name": "Keyword(Iteration)", "fore": "keyword-blue" },
+		{ "name": "Keyword(Jump)", "fore": "keyword-blue" },
```
If you decide to take https://github.com/mono/monodevelop/pull/9606, you should update both the Light and Dark themes so that `Keyword(Jump)` is `keyword-purple`.
